crypto/x509/x509_lu.c

```diff
@@ -403,8 +420,7 @@ X509_OBJECT *X509_OBJECT_new()
     return ret;
 }
 
-
-void X509_OBJECT_free(X509_OBJECT *a)
+static void X509_OBJECT_free_internal(X509_OBJECT *a)
```

Comment:
Perhaps should be `x509_object_free_internal`?

Reply:
Will fix